src/components/api/codenvy-workspace.factory.js

```diff
@@ -237,7 +238,10 @@ class CodenvyWorkspace {
     return promise;
   }
 
-
+  deleteRuntime(workspaceId) {
```

Comment:
I would call this method as stopWorkspace (better name when we browse list of methods even if behind it calls deleteRuntime)